aptos-move/framework/src/natives/dkg_rounding.rs

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cmp::{max, min};
+use aptos_native_interface::{
+    safely_pop_arg, RawSafeNative, SafeNativeBuilder, SafeNativeContext, SafeNativeResult,
+};
+use aptos_types::dkg::real_dkg::rounding::DKGRounding;
+use move_vm_runtime::native_functions::NativeFunction;
+use move_vm_types::{
+    loaded_data::runtime_types::Type,
+    values::{Struct, Value},
+};
+use smallvec::{smallvec, SmallVec};
+use std::collections::vec_deque::VecDeque;
+use num_integer::Integer;
+
+pub fn make_all(
+    builder: &SafeNativeBuilder,
+) -> impl Iterator<Item = (String, NativeFunction)> + '_ {
+    let mut natives = vec![];
+
+    natives.extend([
+        (
+            "rounding_internal",
+            rounding_internal as RawSafeNative,
+        ),
+        (
+            "rounding_v0_internal",
```

Comment:
Can you make sure this doesn't get included as a non-testing native in `assert_no_test_natives` in `aptos-move/aptos-vm/src/natives.rs`?

Reply:
plan to delete it.